### PR TITLE
Playwright Test: Special:RecentChanges should show page creation

### PR DESCRIPTION
**Pull request by hariclerry**

### CODE REVIEW

File: tests/playwright/specs/specialrecentchanges.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+		const content = 'content-Iñtërnâtiônàlizætiøn';
+		const name = 'Name-Iñtërnâtiônàlizætiøn';
+		const bot = await Api.bot();
+
+		const changesList = '.mw-changeslist';
+		const changeListTitles = '.mw-changeslist-title';
+
+		await bot.edit( name, content );
+		await page.goto(
+			`${global.baseUrl}/index.php?${querystring.stringify( 'Special:RecentChanges' )}# `
```
Comment (zeljkofilipin) [inline tests/playwright/specs/specialrecentchanges.js]: I think this should just work:

    await page.goto(`${global.baseUrl}/wiki/Special:RecentChanges`)

**Review by zeljkofilipin — CHANGES_REQUESTED** [review]
Looks good! I've left one inline comment.

File: tests/playwright/specs/specialrecentchanges.js (hunk 1 above)
Comment (zeljkofilipin) [reply tests/playwright/specs/specialrecentchanges.js]: Ok, try this. 😄 

    await page.goto(`${global.baseUrl}/index.php?title=Special:RecentChanges`)

**Review by zeljkofilipin — CHANGES_REQUESTED** [review]
Sorry, wrong code in the previous comment. 🤦

File: tests/playwright/specs/specialrecentchanges.js (hunk 1 above)
Comment (hariclerry) [reply tests/playwright/specs/specialrecentchanges.js]: This has worked. Thanks

**Comment by hariclerry** [comment]
This has worked. Thanks

**Comment by zeljkofilipin** [comment]
Looks good! 🥳

**Pull request merged**